department-of-veterans-affairs/abd-vro
# remove all versioning

## What was the problem?

Implementation of the tech spec which recommends removing versioning and releases.

Associated tickets or Slack threads:
- #2506 

## How does this fix it?[^1]

- Removes all of the logic supporting the automations around release, versioning, and version pinning.

## How to test this PR
- Attempt to deploy using the new workflow definitions on this custom branch

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
